src/decorators.js

```diff
@@ -55,10 +55,15 @@ export function customElement(name: string): any {
 /**
 * Decorator: Indicates that the decorated class is a custom attribute.
 * @param name The name of the custom attribute.
-* @param defaultBindingMode The default binding mode to use when the attribute is bound wtih .bind.
+* @param defaultBindingMode The default binding mode to use when the attribute is bound with .bind.
+* @param aliases The array of aliases to associate to the custom attribute.
 */
-export function customAttribute(name: string, defaultBindingMode?: number): any {
+export function customAttribute(name: string, defaultBindingMode?: number, aliases?: string[]): any {
   return function(target) {
+    if (aliases) {
+      target.aliases = aliases;
```

Comment:
@EisenbergEffect I do understand your point but that does not solve the use case I was describing. If somebody consumes the i18n library, he needs a way to define the aliases for the TCustomAttribute. The consumer has no idea about HtmlBehaviorResources nor metadata nor anything.

Even requiring him to do

```
import {TCustomAttribute} from 'aurelia-i18n'

configure()... {
  TCustomAttribute.aliases = [...]
}
```

isn't the nicest api but its yet understandable. So with the above described metadata approach, how would the consumer of i18n define aliases for the TCustomAttribute?